### PR TITLE
Add MSOutputUrl

### PR DESCRIPTION
**Pull request by todor-ivanov**
This is needed for the newly developed Microservice MSRuleCleaner

### CODE REVIEW

File: reqmgr2ms/config-ruleCleaner.py [hunk 1]
```diff
@@ -69,6 +69,7 @@
 data.couch_wmstats_db = "wmstats"
 data.manager = 'WMCore.MicroService.Unified.MSManager.MSManager'
 data.reqmgr2Url = "%s/reqmgr2" % BASE_URL
+data.MSOutputUrl = "%s/ms-output" % BASE_URL
```
Comment (amaltaro) [inline reqmgr2ms/config-ruleCleaner.py:72]: Todor, we try to stick with lower camel case. So can you please change it to `data.msOutputUrl` instead?

Comment (todor-ivanov) [reply reqmgr2ms/config-ruleCleaner.py:72]: Ok @amaltaro. Just a heads up, I will need to add few more parameters to the service, so I will reuse this PR and will just rename it.

**Comment by amaltaro** [comment]
Please make the necessary changes against gitlab services_config and link them here in the initial description.

**Review by amaltaro — APPROVED** [review]

**Comment by amaltaro** [comment]
For reference, it goes along with:
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/42
and
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/43

**Pull request merged**